services/notify/schemas/v1/notification-address-list.yml

```diff
@@ -0,0 +1,20 @@
+$schema:  "/schemas/common/metaschema.json#"
+title:  "List of notification adresses"
+description:  |
+  List of notification addresses.
```

Comment:
List of *blacklisted* notification addresses

Reply:
I thought to keep it general so it could be used in the future or will that be not needed?

Reply:
Ah, that's a fair point!